source/Calamari/Kubernetes/Commands/KubernetesApplyRawYamlCommand.cs

```diff
@@ -75,70 +54,13 @@ public override int Execute(string[] commandLineArguments)
                 throw new InvalidOperationException(
                     "Unable to execute the Kubernetes Apply Raw YAML Command because the appropriate feature has not been enabled.");
 
```

Comment:
A lot of what is removed below has been moved to here: https://github.com/OctopusDeploy/Calamari/pull/1032/files#r1218793262